Uchu.Core/Handlers/Commands/StandardCommandHandler.cs

```diff
@@ -290,13 +290,20 @@ public static async Task<string> SetGameMasterLevel(string[] arguments)
             {
                 return "Invalid <level>";
             }
+            
+            if (Enum.IsDefined(typeof(GameMasterLevel), int.Parse(arguments[1])))
```

Comment:
Seems redundant to have both this check at line 289:
```cs
if (!Enum.TryParse<GameMasterLevel>(arguments[1], out var level))
{
    return "Invalid <level>";
}
```
and this one:
```cs
if (Enum.IsDefined(typeof(GameMasterLevel), int.Parse(arguments[1])))
{
    // ...
}
else
{
    return "Invalid <level>";
}
```

Reply:
One checks if the number is a int and the other checks if it is in the enum

Reply:
My bad, it's not redundant, I misunderstood how TryParse works. But [TryParse](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0#code-try-6) will also work with the name of the enum value like "Mythran", which will give `level` the correct value, but then fail in the next conditional as the argument "Mythran" can't/won't be parsed as int. So I think that should be `if Enum.IsDefined(typeof(GameMasterLevel), level)`

Reply:
This does seem a bit redundant.
If a number is specified that is outside the valid range [TryParse](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0) will _succeed_, and convert `arguments[1]` to `level`.
So the `int.Parse(arguments[1])` seems redundant, as `level == int.Parse(arguments[1])` if `arguments[1]` is a number.

Also, if the world "Console" is specified, `Int.Parse` would error, which would be an issue.

The two exception cases can both be combined for simplicity.

So I propose this check should be used instead

```c#
if (!Enum.TryParse<GameMasterLevel>(arguments[1], out var level) || 
    !Enum.IsDefined(typeof(GameMasterLevel), level))
{
    return "Invalid <level>";
}

user.GameMasterLevel = (int) level;

...
```

Reply:
An similar case needs to be solve here: https://github.com/UchuServer/Uchu/blob/3421c7079bc4927ec8a538e88b3526c12d30c3ee/Uchu.Master/Api/AccountCommands.cs#L133

To prevent invalid enum values set via the API.